ui/src/components/Activity/ActivityItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import ActivitySearch from "./ActivityItems/ActivitySearch";
+
+class ActivityItem extends Component {
```

Comment:
I'm not sure what does this component do?

Reply:
In the feature we must include more activity types(not only search results) in the `/activity` screen, so `ActivityItem` component is a layer for switching a different types of activities.

PS: At the current moment this component is useless and can be removed, but in long term perspective we need it.